mpeg-encoder.js

```diff
@@ -0,0 +1,58 @@
+module.exports = function () {
+  // eslint-disable-next-line
+  importScripts('https://cdnjs.cloudflare.com/ajax/libs/lamejs/1.2.0/lame.min.js')
+
+  // blob.arrayBuffer() polyfill
+  function arrayBufferPolyfill () {
+    return new Promise(function (resolve) {
+      var fr = new FileReader()
+      fr.onload = function () {
+        resolve(fr.result)
+      }
+      // eslint-disable-next-line
+      fr.readAsArrayBuffer(this);
+      // eslint-disable-next-line
+    }.bind(this))
+  }
+  Blob.prototype.arrayBuffer = Blob.prototype.arrayBuffer || arrayBufferPolyfill
+
+  var CHANNELS = 1
+  var KBPS = 128
+  var SAMPLE_RATE = 44100
+
+  // eslint-disable-next-line
+  var encoder = new lamejs.Mp3Encoder(CHANNELS, SAMPLE_RATE, KBPS)
```

Comment:
What ESLint warning do we have here? Maybe I can recommend way to avoid disabling the linter.

Reply:
undef lamejs

Reply:
You can use `window.lamejs`. It will not have warning.

Reply:
done